distribution/test/itests/test-itests-common/src/main/java/ddf/common/test/catalog/CatalogTestCommons.java

```diff
@@ -0,0 +1,164 @@
+                    .getHeader("id");
+        }
+    }
+
+    public static String ingestGeoJson(String json) {
+        return ingest(json, "application/json");
+    }
+
+    public static Map<String, String> ingestMetacards(Map<String, String> metacardsIds) {
+        //ingest csw
+        String cswRecordId = ingestCswRecord(getFileContent(
+                CSW_RESOURCE_ROOT + "csw/record/CswRecord.xml"));
+        metacardsIds.put(CSW_METACARD, cswRecordId);
+
+        //ingest xml
+        String plainXmlNearId = ingest(getFileContent(
+                CSW_RESOURCE_ROOT + "xml/PlainXmlNear.xml"), MediaType.TEXT_XML);
+        String plainXmlFarId = ingest(getFileContent(
+                CSW_RESOURCE_ROOT + "xml/PlainXmlFar.xml"), MediaType.TEXT_XML);
+        metacardsIds.put(PLAINXML_NEAR_METACARD, plainXmlNearId);
+        metacardsIds.put(PLAINXML_FAR_METACARD, plainXmlFarId);
+
+        //ingest json
+        String geoJsonNearId = ingestGeoJson(getFileContent(
+                CSW_RESOURCE_ROOT + "json/GeoJsonNear.json"));
+        String geoJsonFarId = ingestGeoJson(getFileContent(
+                CSW_RESOURCE_ROOT + "json/GeoJsonFar.json"));
+        metacardsIds.put(GEOJSON_NEAR_METACARD, geoJsonNearId);
+        metacardsIds.put(GEOJSON_FAR_METACARD, geoJsonFarId);
+
+        return metacardsIds;
+    }
+
+    public static String ingestCswRecord(String cswRecord) {
+
+        String transactionRequest = getCswInsert("csw:Record", cswRecord);
+
+        ValidatableResponse response = given().log()
+                .all()
+                .body(transactionRequest)
+                .header("Content-Type", MediaType.APPLICATION_XML)
+                .when()
+                .post(CSW_PATH.getUrl())
+                .then()
+                .log()
+                .all()
+                .assertThat()
+                .statusCode(equalTo(HttpStatus.SC_OK));
+
+        return response.extract()
+                .body()
+                .xmlPath()
+                .get("Transaction.InsertResult.BriefRecord.identifier")
+                .toString();
+    }
+
+    public static void update(String id, String data, String mimeType) {
+        given().header(HttpHeaders.CONTENT_TYPE, mimeType)
+                .body(data)
+                .expect()
+                .log()
+                .all()
+                .statusCode(HttpStatus.SC_OK)
+                .when()
+                .put(new AbstractIntegrationTest.DynamicUrl(REST_PATH, id).getUrl());
+    }
+
+    public static void deleteMetacard(String id) {
+        deleteMetacard(id, true);
+    }
+
+    public static void deleteMetacard(String id, boolean checkResponse) {
+        if (checkResponse) {
+            delete(REST_PATH.getUrl() + id).then()
+                    .assertThat()
+                    .statusCode(200)
+                    .log()
+                    .all();
+        } else {
+            delete(REST_PATH.getUrl() + id).then()
```

Comment:
Do we need to assert the status code in this else clause and remove the log all. Also, same comment on changing `checkResponse` to `logResponse` in method signature.